.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     branches:
-      - master
+      - '*'
```

Comment:
This looks like it will run workflows twice now. The benefit prior was master always got a build (after a merge), while the original pull request only got a workflow built via the `pull_request` path.